definitions/variable/energy/secondary-energy-electricity.yaml

```diff
@@ -0,0 +1,41 @@
+- Secondary Energy|Electricity:
+    definition: Total net electricity generation
+    unit: EJ/yr
+
+- Secondary Energy|Electricity|Biomass:
+    definition: Net electricity generation from purpose-grown bioenergy crops, crop and
+      forestry residue, municipal solid waste bioenergy and traditional biomass
+    unit: EJ/yr
+- Secondary Energy|Electricity|Biomass|{CCS}:
+    definition: Net electricity generation from purpose-grown bioenergy crops, crop and
+      forestry residue, municipal solid waste bioenergy and traditional biomass {CCS}
+    unit: EJ/yr
+
+- Secondary Energy|Electricity|Non-Biomass Renewables:
+    definition: Net electricity generation from hydro, wind, solar, geothermal, ocean,
+      and other renewable sources excluding biomass
+    unit: EJ/yr
+- Secondary Energy|Electricity|{Non-Biomass Renewables}:
```

Comment:
Do I understand that correctly that while `Secondary Energy|Electricity|Wind` is a subcategory of `Secondary Energy|Electricity|Non-Biomass Renewables`, it is not filed as `Secondary Energy|Electricity|Non-Biomass Renewables|Wind`. In the AR6 mapping file, we had sometimes both (for example [`Primary Energy|Non-Biomass Renewables|Wind`](https://github.com/pik-piam/piamInterfaces/blob/6209d8f52cf04a40cb5d1b2c67bae04e9ba3fe7c/inst/templates/mapping_template_AR6.csv#L1191) and [`Primary Energy|Wind`](https://github.com/pik-piam/piamInterfaces/blob/6209d8f52cf04a40cb5d1b2c67bae04e9ba3fe7c/inst/templates/mapping_template_AR6.csv#L1207) and that lead to some confusion, I think.

Reply:
Correct, there was some confusion between trying to have a strictly hierarchical variable tree (Non-Biomass Renewables -> Wind) versus a more concise, readable approach (Wind).

Generally, the consensus in the community is (as far as I am aware) to prefer readability and have the additional information (which categories are part of e.g. Non-Biomass Renewables) either implicit or explicit (as an attribute of the definitions-yaml).

Reply:
As you said that we will have explicit summation definitions and checks, that should cover these cases.